src/Admin/Controllers/ToolsController.cs

```diff
@@ -176,5 +192,77 @@ public async Task<IActionResult> PromoteAdmin(PromoteAdminModel model)
             await _organizationUserRepository.ReplaceAsync(user);
             return RedirectToAction("Edit", "Organizations", new { id = model.OrganizationId.Value });
         }
+
+        public IActionResult GenerateLicense()
+        {
+            return View("GenerateLicense");
```

Comment:
Is the name of the view required here if it matches the name of the action method?